Public/Sdk/UnitTests/Transformers/Test.write/writeAllLines.lkg

```diff
@@ -0,0 +1,14 @@
+Transformer.writeFile(
+    f`./Out/file`,
+    {
+        escaping: "NoEscaping",
+        separator: "",
+        items: [
+            {
+                escaping: "NoEscaping",
+                separator: "\r\n",

```

Comment:
Just curious if `writeAllLines` uses `\r\n` as the separator even on platforms where `Environment.NewLine` is something else (no action needed here)

Reply:
It uses current platform. See: [http://index/?query=ambienttransformer&rightProject=BuildXL.FrontEnd.Script&file=Ambients%5CTransformers%5CAmbientTransformer.Write.cs&line=102](http://index/?query=ambienttransformer&rightProject=BuildXL.FrontEnd.Script&file=Ambients%5CTransformers%5CAmbientTransformer.Write.cs&line=102)

---
In reply to: [272256754](https://github.com/Microsoft/BuildXL/pull/82#discussion_r272256754) [](ancestors = 272256754)